internal/mode/static/metrics/collectors/nginx.go

```diff
@@ -13,13 +16,45 @@ import (
 )
 
 const (
-	nginxStatusSock = "/var/run/nginx/nginx-status.sock"
-	nginxStatusURI  = "http://config-status/stub_status"
+	nginxStatusSock        = "/var/run/nginx/nginx-status.sock"
+	nginxStatusURI         = "http://config-status/stub_status"
+	nginxPlusAPISock       = "/var/run/nginx/nginx-plus-api.sock"
+	nginxPlusAPIURI        = "http://nginx-plus-api/api"
+	nginxStatusSockTimeout = 10 * time.Second
+)
+
+var (
+	upstreamServerVariableLabels = []string{
```

Comment:
since we're not populating those labels (NGF doesn't set them based on the cluster state), do we really need them?

Prometheus endpoint output:
```
nginx_gateway_fabric_upstream_server_response_time{class="nginx",pod_name="",resource_name="",resource_namespace="",resource_type="",server="10.244.0.6:8080",service="",upstream="default_coffee_80"} 0
nginx_gateway_fabric_upstream_server_response_time{class="nginx",pod_name="",resource_name="",resource_namespace="",resource_type="",server="10.244.0.7:8080",service="",upstream="default_tea_80"} 0
```

Reply:
Good point. I'll remove them.

Reply:
looks like this wasn't removed yet